clang/test/CodeGenSYCL/intel-fpga-loops.cpp

```diff
@@ -134,7 +126,6 @@ void speculated_iterations() {
       a[i] = 0;
 }
 
-// Add CodeGen tests for FPGA loop attribute: [[intel::fpga_pipeline()]].
```

Comment:
This comment was just misplaced.
Suggest changing this to:
`// Add CodeGen tests for FPGA loop_count attributes.`
instead of removing it.